src/tutorials/0006/02.md

```diff
@@ -0,0 +1,35 @@
+By using a cryptographic algorithm to generate the hash, we guarantee that we can identify a file with a single hash.
+
+Occassionally, a hashing algorithm may stop being compliant with the previously stated rules, as has already happened with `sha1`. With time, other algorithms may prove to be insufficient for content addressing in IPFS. For this reason, and in order to support multiple cryptographic algorithms, **we need to be able to know which algorithm was used to generate the hash** of specific content.
```

Comment:
There were called characteristics earlier, so using the same word here for consistency. Also "stop being compliant" suggests it has a choice in the matter 😂.

```suggestion
Occassionally, a hashing algorithm may be proven to be insecure, meaning it no longer complies with the characteristics that we defined earlier. It has already happened with `sha1`. With time, other algorithms may prove to be insufficient for content addressing in IPFS. For this reason, and in order to support multiple cryptographic algorithms, **we need to be able to know which algorithm was used to generate the hash** of specific content.
```